cmd/tester/cmd/ibc_transfer.go

```diff
@@ -0,0 +1,176 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/b-harvest/cosmos-module-stress-test/client"
+	"github.com/b-harvest/cosmos-module-stress-test/config"
+	"github.com/b-harvest/cosmos-module-stress-test/tx"
+	"github.com/b-harvest/cosmos-module-stress-test/wallet"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagPacketTimeoutHeight    = "packet-timeout-height"
+	flagPacketTimeoutTimestamp = "packet-timeout-timestamp"
+	flagAbsoluteTimeouts       = "absolute-timeouts"
+)
+
+func IbctransferCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "transfer [src-port] [src-channel] [receiver] [amount] [round] [tx-num] [msg-num]",
+		Short:   "Transfer a fungible token through IBC",
+		Aliases: []string{"t"},
+		Args:    cobra.ExactArgs(7),
+		Long: `Transfer a fungible token through IBC.
+
+Example: $tester t transfer channel-0 cosmos1pacc0fr45hggcn8jrfhgnqf8vgyqna7r5sftql 10uatom 10 1 1
+
+[round]: how many rounds to run
+[tx-num]: how many transactions to be included in one round
```

Comment:
Add explanation about what `msg-num` does. I understand that this comment must be copied from other CLI commands, but I made modification today. Reference [this swap cli exaplanation](https://github.com/b-harvest/cosmos-module-stress-test/blob/main/cmd/tester/cmd/swap.go#L31).